worlds/animal_well/options.py

```diff
@@ -239,5 +249,6 @@ class AnimalWellOptions(PerGameCommonOptions):
         "obscure_tricks": ObscureTricks.option_true,
         "precise_tricks": PreciseTricks.option_true,
         "tanking_damage": TankingDamage.option_true,
+        "weird_tricks": WeirdTricks.option_true,
```

Comment:
Presets are website-only things, so no need to include the backup weird tricks option there